staff_features/resources/step_definitions/resource_shared.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+When 'the user filters by text with the Resource title' do
+  expect(find('h2').text).to eq 'Resources'
+
+  fill_in 'Filter by text', with: @uuid
+  find('.search-filter button').click
+
+  rows = []
+  checks = 0
+
+  while checks < 5
+    checks += 1
+
+    begin
+      rows = all('tr', text: "Resource #{@uuid}")
+    rescue Selenium::WebDriver::Error::JavascriptError
+      sleep 1
+    end
+
+    break if rows.length == 1
+  end
+end
+
+Then 'the Resource is in the search results' do
+  expect(page).to have_css('tr', text: @uuid)
+end
+
+Then 'the Resource view page is displayed' do
+  expect(find('h2').text).to eq "Resource #{@uuid} Resource"
```

Comment:
@thimios what do you think about this pattern of expecting a <h2>TITLE</h2> as a test for being on a view page? It's easy, and sometimes makes decent sense, for example testing for being on the Resources `index` view like line 48 in the [resources_delete.rb above](https://github.com/archivesspace/e2e-tests/pull/41/files#diff-3654ff69d39e6b173f6fa52a6e43ebadcdf90a7e4d6dc6d9ad77a9ecb6a97e64R48).

There are lots of this pattern in the existing tests, and they are already found commonly here in the e2e project. While I've requested a change to this pattern in another e2e PR, perhaps we just keep using the pattern?